### PR TITLE
chore: Clean up node package scripts

### PR DESCRIPTION
**Pull request by HoonBaek**
## Description Of Changes

* "rollup" and "build" are duplicated, so use "rollup" in "build"
* Modify the "predeploy" to fit the current process

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [ ] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [x] Improvement (refactor code)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
     "start": "rollup -c -w",
-    "reset": "rm -rf ./node_modules; npm install;",
+    "reset": "rm -rf ./node_modules; npm install",
     "prepublishOnly": "npm run build",
-    "predeploy": "cd example && npm install && npm run build",
+    "predeploy": "rm -rf ./release; rm -rf ./dist; npm install; npm run build",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "eslint": "eslint --no-eslintrc  -c .eslintrc.js 'src/**'",
-    "stylelint": "stylelint 'src/**'",
```
Comment (sravan-s) [inline package.json:31]: this change is unnecessary, no?

File: package.json [hunk 2]
```diff
@@ -20,17 +20,17 @@
...
     "lint": "npm run eslint; npm run eslint-ts",
+    "eslint": "eslint --no-eslintrc  -c .eslintrc.js 'src/**'",
     "eslint-ts": "eslint --no-eslintrc -c .eslintrc.json './src/**/*.ts*'",
+    "stylelint": "stylelint 'src/**'",
     "generate-component": "plop",
     "rollup": "npm run clean-release; rollup -c",
```
Comment (sravan-s) [inline package.json:35]: just remove this and keep build as it was

Comment (HoonBaek) [reply package.json:35]: ok ok

File: package.json (hunk 1 above)
Comment (HoonBaek) [reply package.json:31]: yeah unnecessary

Comment (HoonBaek) [reply package.json:31]: I just changed to improve readability

**Review by sravan-s — APPROVED** [review]

**Pull request merged**